slf4j/src/main/resources/META-INF/native-image/ch.qos.logback/logback-classic/reflect-config.json

```diff
@@ -0,0 +1,194 @@
+[
```

Comment:
Why would Pekko need this? Shouldn't this be in logback-classic jar?

If it doesn't matter what, these json files go in then, we could just have standalone jars that have these files instead of having the Pekko jars contain then.

Reply:
logback-classic is an optional dependency - with slf4j, you should be able to add your own compliant logger jar